R/build.r

```diff
@@ -353,8 +358,8 @@ build_site_local <- function(pkg = ".",
     override = override,
     preview = FALSE
   )
-  build_articles(pkg, lazy = lazy, override = override, preview = FALSE)
-  build_tutorials(pkg, override = override, preview = FALSE)
+  build_articles(pkg, lazy = lazy, override = override, preview = FALSE,
+                 toc_depth = toc_depth)
```

Comment:
Did you mean for the `build_tutorials` call to disappear?

Reply:
Ah no, ofc not, this was by accident. Thanks for catching this!